src/app/containers/ConsentBanner/CanonicalLogic/index.js

```diff
@@ -1,10 +1,10 @@
 import Cookie from 'js-cookie';
+import setCookie from '#lib/utilities/setCookie';
 import setCookieOven from './setCookieOven';
 
 const PRIVACY_COOKIE = 'ckns_privacy';
 const EXPLICIT_COOKIE = 'ckns_explicit';
 const POLICY_COOKIE = 'ckns_policy';
-const COOKIE_EXPIRY = 365;
```

Comment:
`COOKIE_EXPIRY` should stay here with the cookie and be passed in as an argument. This is because the cookie banner should have a value of 365 days, but the Script should have a value of 7 days to match the current implementation on other pages.

Reply:
I've set this as the default now in `setCookie` [here](https://github.com/bbc/simorgh/blob/2313e38da44f9972c6690a1eceaa2b72890a84c1/src/app/lib/utilities/setCookie/index.jsx#L16).